packages/wallet/src/services/TransactionsTracker.ts

```diff
@@ -217,12 +220,13 @@ export const createTransactionsTracker = (
     tip$,
     chainHistoryProvider,
     addresses$,
-    newTransactions: { submitting$: newSubmitting$, pending$, failedToSubmit$ },
+    newTransactions: { submitting$: newSubmitting$, pending$, signed$: newSigned$, failedToSubmit$ },
```

Comment:
We should store signed transactions in pouchdb, just like we do for in-flight transactions
```suggestion
    newTransactions: { submitting$: newSubmitting$, pending$, signed$, failedToSubmit$ },
    ...
    // should be similar to submittingOrPreviouslySubmitted$
    const newSigned$ = ...
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/cdbad467fb4a95811c6ca9c5c68bdb3c82b5a417